### PR TITLE
stock status of duplicated product

### PR DESCRIPTION
**Pull request by vovsky**
### Fixed Issues (if relevant)
1. magento-engcom/msi#666: When we apply "Save & Duplicate" for existing Simple product - stock status for new product should be "Out of stock"

### Manual testing scenarios
1. Create simple product 
2. Press Save & Duplicate button
3. Check Stock Status of duplicated product

### CODE REVIEW

**Comment by larsroettig** [comment]
Hi @vovsky, thank you for contribution. Please Fix the Static Test see:
https://travis-ci.org/magento-engcom/msi/jobs/357811224

Best regards,
Lars

File: app/code/Magento/InventoryCatalog/Model/UpdateSourceItemBasedOnLegacyStockItem.php [hunk 1]
```diff
@@ -105,6 +106,10 @@ public function execute(Item $legacyStockItem)
             $sourceItem->setSku($productSku);
         }
 
+        if ($legacyStockItem->getData(StockItemInterface::USE_CONFIG_MANAGE_STOCK) === null){
+            $legacyStockItem->setData(StockItemInterface::USE_CONFIG_MANAGE_STOCK, 1);
+        }
+
```
Comment (larsroettig) [inline app/code/Magento/InventoryCatalog/Model/UpdateSourceItemBasedOnLegacyStockItem.php]: static test are red

**Review by larsroettig — CHANGES_REQUESTED** [review]
Hi @vovsky, thank you for contribution. Please Fix the Static Test see:
https://travis-ci.org/magento-engcom/msi/jobs/357811224

Best regards,
Lars

**Comment by vovsky** [comment]
@larsroettig please review

**Comment by larsroettig** [comment]
Looks good but a **Integration test** will improve the quality  😸 maybe you have time to write one

**Review by larsroettig — APPROVED** [review]

**Pull request closed without merging**